### PR TITLE
Feature/staging improvements

### PR DESCRIPTION
**Pull request by eskilsen**
From testing in stage we found several bugs that has been improved before the release of v. 8
Mostly improvements on documentation. 

## Motivation and Context

Make sure our DG is the best as possible before release.
## How Has This Been Tested?

Ran all existing tests and update snapshots.
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/App/components/Sheet/index.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ const Sheet = ({ id, requireAction, title, footer, children }) => (
             </div>
             {footer &&
                 <div className="sheet-footer">{"\n"}
-                    <Button type="primary" className="mr-2" label="Primary" />{"\n"}
-                    <Button type="secondary" className="ml-2" label="Secondary" />{"\n"}
+                    <button className="btn ml-2 btn-primary" data-sheet-close={id} type="button">Primary</button>{"\n"}
```
Comment (eskilsen) [inline src/App/components/Sheet/index.js:20]: Need to switch ml-1 and mr-1 😄

Comment (PhungNg) [reply src/App/components/Sheet/index.js:20]: Nice catch!

File: src/App/components/Alert/index.js [hunk 2]
```diff
@@ -14,9 +14,9 @@ const Alert = ({ id, type, icon, close, text, className }) => {
             <i className="material-icons alert-icon" aria-hidden="true">{icon}</i>
             {text}{"\n"}
             {close ? <>
-                <a href="#" data-alert-close="" aria-label="Close alert" onClick={e => e.preventDefault()}>{"\n\t\t"}
+                <button data-alert-close="" onClick={e => e.preventDefault()}>{"\n\t\t"}
```
Comment (eskilsen) [inline src/App/components/Alert/index.js:17]: This should maybe have aria-label to <button> after all?? 🤔

Comment (PhungNg) [reply src/App/components/Alert/index.js:17]: The icon element does not have an aria-hidden attribute, and the icon name is "close". Is it then necessary?

File: src/less/components/alert.less [hunk 3]
```diff
@@ -28,6 +28,8 @@
     }
 
     [data-alert-close] {
+        all: unset;
```
Comment (eskilsen) [inline src/less/components/alert.less:31]: What does this do? :)

File: src/App/components/Alert/index.js (hunk 2 above)
Comment (eskilsen) [reply src/App/components/Alert/index.js:17]: Probably not 👍

**Review by eskilsen — COMMENTED** [review]
Boom boom! Small change required 👍

File: src/less/components/alert.less (hunk 3 above)
Comment (eskilsen) [reply src/less/components/alert.less:31]: Found out 🔍

Comment (PhungNg) [reply src/less/components/alert.less:31]: It removes the button styling

File: src/App/components/Alert/index.js (hunk 2 above)
Comment (eskilsen) [reply src/App/components/Alert/index.js]: RESOLVED

**Review by PhungNg — APPROVED** [review]
Good job to us, brother! 👬

**Pull request merged**